frontend/src/scenes/userLogic.ts

```diff
@@ -89,6 +91,12 @@ export const userLogic = kea<userLogicType>([
                 }),
             },
         ],
+        globalSessionFilters: [
```

Comment:
🤔 longer term I think this lives better off the user. insight logics could store this into a shared location in storage and read/clear that as appropriate.

Or it becomes a team setting... "turn this on for all" like with local account/test filters

But no reason to block this here!

Reply:
yeah wanted to have it on insight logic at first but need something that persists across the session. Local/session storage could be the way to go